hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearchindexer/integration/resource/CompareIndexResourceApiTest.kt

```diff
@@ -13,12 +14,16 @@ import org.mockito.kotlin.eq
 import org.mockito.kotlin.isNull
 import org.mockito.kotlin.timeout
 import org.mockito.kotlin.verify
+import org.springframework.boot.test.mock.mockito.SpyBean
 import uk.gov.justice.digital.hmpps.prisonersearch.model.SyncIndex
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.integration.IntegrationTestBase
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.integration.PrisonerBuilder
 import uk.gov.justice.digital.hmpps.prisonersearchindexer.integration.wiremock.PrisonApiExtension.Companion.prisonApi
 
 class CompareIndexResourceApiTest : IntegrationTestBase() {
+  @SpyBean
+  lateinit var telemetryClient: TelemetryClient
```

Comment:
moved down into only class using the bean so that we will get errors in the tests if the bean isn't defined